core-services/prow/02_config/open-cluster-management-io/policy-generator-plugin/_prowconfig.yaml

```diff
@@ -9,6 +9,7 @@ branch-protection:
               protect: true
               required_pull_request_reviews:
                 dismiss_stale_reviews: true
+                required_approving_review_count: 1
```

Comment:
Did you mean to change this too?

Reply:
Yes--we want to gate upstream PRs with reviews.